resources/install.rb

```diff
@@ -92,27 +102,78 @@
         options '--force-confdef --force-confold'
         action :install
       end
+    elsif node.platform_family? 'windows'
+
+      service "telegraf_#{new_resource.name}" do
+        service_name 'telegraf'
+        action [:stop]
+        only_if { ::Win32::Service.exists?('telegraf') }
+      end
+
+      file_name = "telegraf-#{new_resource.install_version}_windows_amd64.zip"
+      remote_file "#{Chef::Config[:file_cache_path]}/#{file_name}" do
+        source "#{node['telegraf']['download_urls']['windows']}/#{file_name}"
+        checksum node['telegraf']['shasums']['windows']
+        action :create
+      end
+
+      windows_zipfile ENV['ProgramW6432'] do
+        source "#{Chef::Config[:file_cache_path]}/#{file_name}"
+        not_if { ::File.exist?("#{ENV['ProgramW6432']}\\telegraf\\telegraf.exe") }
+        action :unzip
+      end
+
+      directory "#{ENV['ProgramW6432']}\\telegraf\\telegraf.d" do
+        action :create
+      end
+
+      windows_package 'telegraf' do
+        source "#{ENV['ProgramW6432']}\\telegraf\\telegraf.exe"
+        options "--service install --config-directory \"#{ENV['ProgramW6432']}\\telegraf\\telegraf.d\""
+        # rubocop:enable Metrics/LineLength
```

Comment:
You're missing the rubocop disable here

Reply:
yeah forgot that. cookstyle does not peep about that. Yet rubocop does. I will just add them to be sure.